app/assets/javascripts/lookup_keys.js

```diff
@@ -7,6 +7,11 @@ $(function() {
   $('.tabs-left .col-md-4').removeClass('col-md-4').addClass('col-md-8')
   //remove variable click event
   $(document).on('click', '.smart-var-tabs li a span', function(){ remove_node(this);});
+  fill_in_matchers();
+  $('#smart_class_param').parents('form').on('submit', function(){
+    build_match();
+  })
+  $('.matcher_key').select2('destroy');
```

Comment:
@shlomizadok this is not destroying the select2 and when using select2 and adding to the options I get a list of simple text instead of the option list